Products/CMFPlone/controlpanel/browser/recyclebin.py

```diff
@@ -0,0 +1,40 @@
+# testcp.py
+from plone.app.registry.browser.controlpanel import ControlPanelFormWrapper
+from plone.app.registry.browser.controlpanel import RegistryEditForm
+from plone.z3cform import layout
+from zope import schema
+from zope.interface import Interface
+
+
+class IRecycleBinControlPanelSettings(Interface):
+    recycling_enabled = schema.Bool(
+        title="Enable the Recycle Bin",
+        description="Enable or disable the Recycle Bin feature.",
+        default=False,
+    )
+
+    retention_period = schema.Int(
+        title="Retention Period",
+        description="Number of days to keep deleted items in the Recycle Bin. Set to 0 to disable automatic purging.",
+        default=30,
+        min=0,
+    )
+
+    maximum_size = schema.Int(
+        title="Maximum Size",
+        description="Maximum size of the Recycle Bin in MB.",
```

Comment:
```suggestion
        title="Maximum size",
        description="Maximum size of the recycle bin in MB. When the total size of items in the recycle bin exceeds its maximum size, the oldest items in the recycle bin will be permanently purged.",
```

Reply:
this is already done 
![image](https://github.com/user-attachments/assets/f81b007a-72e5-4424-b921-05afa3f621f4)